point-of-care-poc/installation-and-configuration/roles.md

```diff
@@ -0,0 +1,19 @@
+| Personnel               | Role                       | Location    | Access To   |
+|-------------------------|----------------------------|-------------|-------------|
```

Comment:
@slubwama Can you add a section for locations and describe what actions happen at each location, the business rules, etc

Reply:
@ssmusoke maybe in another PR